src/components/accessories/billFilterForm/consts.ts

```diff
@@ -0,0 +1,33 @@
+import { TFields } from "../../../libraries/formDataHandling/types";
+import { BillFilterFormFieldName } from "./types";
+
+export const initialFields: TFields<BillFilterFormFieldName> = {
+  user: {
+    value: "",
+    type: "text",
+    options: [],
+  },
+  patient: {
+    value: "",
+    type: "text",
+    options: [],
+  },
+  billItem: {
+    value: "",
+    type: "text",
+    options: [],
+  },
+  affiliate: {
+    value: "",
+    type: "text",
+    options: [],
+  },
+  fromDate: {
+    value: new Date().setHours(0) + "",
+    type: "date",
+  },
+  toDate: {
+    value: new Date().setHours(23) + "",
```

Comment:
There is a reason why you are using setHours?

Reply:
Yes. For filter purpose I set the time to 23 in order extend the time period.

Reply:
Ok, but why? I think that the date at 24pm should be enought, right? I don't understand why you decrease the hours to 23